membership-common/src/main/scala/com/gu/zuora/ZuoraSoapService.scala

```diff
@@ -88,6 +104,16 @@ class ZuoraSoapService(soapClient: soap.ClientWithFeatureSupplier)(implicit ec:
     )
   }
 
+  // Creates a payment method in zuora and sets it as default in the specified account.To satisfy zuora validations this has to be done in three steps
+  def createPaymentMethod(command: CreatePaymentMethod)(implicit logPrefix: LogPrefix): Future[UpdateResult] = for {
+    _ <- setGatewayAndClearDefaultMethod(
+      command.accountId,
+      command.paymentGateway,
+    ) // We need to set gateway correctly because it must match with the payment method we'll create below
+    createMethodResult <- soapClient.extendedAuthenticatedRequest[CreateResult](new XmlWriterAction(command)(createPaymentMethodWrites))
+    result <- setDefaultPaymentMethod(command.accountId, createMethodResult.id, command.paymentGateway, command.invoiceTemplateOverride)
+  } yield result
+
```

Comment:
had to add back in to membership-common methods that were actually used in the copied version